src/kaleidoscope/plugin/LEDControl.cpp

```diff
@@ -112,21 +112,24 @@ void LEDControl::set_all_leds_to(uint8_t r, uint8_t g, uint8_t b) {
 }
 
 void LEDControl::set_all_leds_to(cRGB color) {
-  for (int8_t i = 0; i < LED_COUNT; i++) {
-    setCrgbAt(i, color);
+  for (int8_t led_index = 0; led_index < LED_COUNT; led_index++) {
+    setCrgbAt(led_index, color);
```

Comment:
Please don't mix variable renamings in with implementation changes.

Reply:
That slipped my final check. Will be reverted.